sql/src/main/java/io/crate/execution/engine/window/WindowProjector.java

```diff
@@ -99,16 +99,16 @@ public static WindowProjector fromProjection(WindowAggProjection projection,
             FunctionImplementation impl = functions.getQualified(
                 windowFunctionContext.function().info().ident());
             if (impl instanceof AggregationFunction) {
-                ExpressionsInput<Row, Boolean> filter;
-
-                if (windowFunctionContext.filter().symbolType() == SymbolType.LITERAL) {
```

Comment:
this is not needed. InputFactory visits the literal symbol. Also it was not correct.